.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/*
+node_modules/
```

Comment:
As this is a ruby/rails project. Please add logfiles, tempfiles and other rails related files not needed to be committed.